spec/system/admin/orders_spec.rb

```diff
@@ -584,6 +584,23 @@ def extract_pdf_content
             reader.pages.map(&:text)
           end
 
+          def print_all_invoices
+            page.find("span.icon-reorder", text: "ACTIONS").click
+            within ".ofn-drop-down .menu" do
+              expect {
+                page.find("span", text: "Print Invoices").click # Prints invoices in bulk
+              }.to enqueue_job(BulkInvoiceJob).exactly(:once)
+            end
+
+            expect(page).to have_content "Compiling Invoices"
+            expect(page).to have_content "Please wait until the PDF is ready " \
+                                         "before closing this modal."
+
+            perform_enqueued_jobs(only: BulkInvoiceJob)
+
+            expect(page).to have_content "Bulk Invoice created"
+          end
```

Comment:
I would put these methods at the end of the file, just before the last `end` line.